dotcom-rendering/src/web/components/FrontPage.tsx

```diff
@@ -52,6 +53,9 @@ export const FrontPage = ({ front, NAV }: Props) => {
 			<Island clientOnly={true} deferUntil="idle">
 				<FetchCommentCounts repeat={true} />
 			</Island>
+			<Island clientOnly={true} deferUntil="idle">
```

Comment:
Will deferUntil="idle" add to CLS? Wasn't sure if you wanted to try expediteLoading={true} instead?

Reply:
I do! Lets give it a go!